airflow/models/baseoperator.py

```diff
@@ -190,42 +191,42 @@ def partial(
     task_id: str,
     dag: DAG | None = None,
     task_group: TaskGroup | None = None,
-    start_date: datetime | None = None,
-    end_date: datetime | None = None,
-    owner: str = DEFAULT_OWNER,
-    email: None | str | Iterable[str] = None,
+    start_date: datetime | None | ArgNotSet = NOTSET,
+    end_date: datetime | None | ArgNotSet = NOTSET,
+    owner: str | None | ArgNotSet = NOTSET,
```

Comment:
I don’t think the None case is possible here? (Also `trigger_rule` and maybe more below)